Global.h

```diff
@@ -157,6 +157,7 @@ static_assert(sizeof(bool) == 1, "Bool needs to be 1 byte in size.");
 #include <boost/thread.hpp>
 #include <boost/variant.hpp>
 #include <boost/math/special_functions/round.hpp>
+#include <boost/multi_array.hpp>

```

Comment:
GCC already can`t use our precompiled header - just crushes. We should cleanup Global.h ant not add more heavy stuff there.